src/cata_tiles.cpp

```diff
@@ -270,6 +270,11 @@ void cata_tiles::load_tilejson_from_file(std::ifstream &f, const std::string &im
         JsonObject curr_info = info.next_object();
         tile_height = curr_info.get_int("height");
         tile_width = curr_info.get_int("width");
+        if (curr_info.has_bool("iso")) {
```

Comment:
Defaults can also be dangerous as the program can not detect spelling errors that way. They are useful if there is a meaningful default (e.g. a value that is used in nearly all situations), or (like here) for backwards compatibility.

But this is a different discussion, my comment was about syntax: if you have a default value, use the json function that will automatically use them. Don't repeat the "if value does not exist, return default" code, which is already part of the json system. This is also easier to read.